specification/iotsecurity/resource-manager/Microsoft.IoTSecurity/preview/2021-02-01-preview/sensors.json

```diff
@@ -0,0 +1,507 @@
+        "description": "Trigger threat intelligence package update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SensorScope"
+          },
+          {
+            "$ref": "#/parameters/SensorName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Triggered threat intelligence package update"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SensorsList": {
+      "description": "List of IoT sensors",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List data",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SensorsModel"
+          }
+        }
+      }
+    },
+    "SensorsModel": {
+      "type": "object",
+      "description": "IoT sensor model",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "IoT sensor properties",
+          "$ref": "#/definitions/SensorProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "SensorProperties": {
+      "type": "object",
+      "description": "IoT sensor properties",
+      "properties": {
+        "connectivityTime": {
+          "description": "Last connectivity time of the IoT sensor",
+          "readOnly": true,
+          "type": "string",
+          "example": "2020-11-17T12:31:25Z"
+        },
+        "creationTime": {
+          "description": "Creation time of the IoT sensor",
+          "readOnly": true,
+          "type": "string",
+          "example": "2020-11-17T12:31:25Z"
+        },
+        "dynamicLearning": {
+          "description": "Dynamic mode status of the IoT sensor",
+          "example": true,
+          "readOnly": true,
+          "type": "boolean"
+        },
+        "learningMode": {
+          "description": "Learning mode status of the IoT sensor",
+          "example": true,
+          "readOnly": true,
+          "type": "boolean"
+        },
+        "sensorStatus": {
+          "readOnly": true,
+          "type": "string",
+          "example": "Ok",
+          "description": "Status of the IoT sensor",
+          "enum": [
+            "Ok",
+            "Disconnected",
+            "Unavailable"
+          ],
+          "x-ms-enum": {
+            "name": "sensorStatus",
+            "modelAsString": true,
+            "values": [
```

Comment:
These match the values above, so no reason to specify values, unless you were planning on adding descriptions for each value